server/cmwell-it/src/it/scala/cmwell/it/APIFunctionalityTests.scala

```diff
@@ -664,6 +664,44 @@ class APIFunctionalityTests extends AsyncFunSpec
         }
       }
 
+      describe("cmwell://meta/sys#markReplace predicate with a wildcard") {
+        it("should ingest multiple fields of two ontologies") {
+          val newData =
+            """
+              |<http://example.org/JaneSmith8> <http://www.w3.org/2006/vcard/ns#GENDER> "Female" .
+              |<http://example.org/JaneSmith8> <http://www.w3.org/2006/vcard/ns#FN> "Jane" .
+              |<http://example.org/JaneSmith8> <http://www.w3.org/2006/vcard2/ns#GENDER> "Female" .
+            """.stripMargin
+          Http.post(_in, newData, None, List("format" -> "ntriples"), tokenHeader).map { res =>
+            withClue(res) {
+              jsonSuccessPruner(Json.parse(res.payload)) should be(jsonSuccess)
+            }
+          }
+        }
+        it("should use markReplace with wildcard and verify it removes all relevant fields") {
+          spinCheck(100.millis, true)(Http.get(exampleOrg./("JaneSmith8"), List("format" -> "json"))) { res =>
```

Comment:
Get is from cassandra. Is this enough for the test?

Reply:
Yes